packages/eslint-config/package.json

```diff
@@ -27,29 +31,35 @@
         }
     },
     "dependencies": {
-        "eslint": "^9.27.0",
-        "eslint-plugin-import": "^2.31.0",
+        "eslint": "^9.39.1",
+        "eslint-plugin-import": "^2.32.0",
         "eslint-plugin-lit": "^2.1.1",
         "eslint-plugin-react": "^7.37.5",
-        "eslint-plugin-react-hooks": "^7.0.0",
-        "eslint-plugin-wc": "^3.0.1"
+        "eslint-plugin-react-hooks": "^7.0.1",
+        "eslint-plugin-wc": "^3.0.2"
     },
     "devDependencies": {
-        "@goauthentik/prettier-config": "^3.1.0",
-        "@goauthentik/tsconfig": "^1.0.1",
+        "@goauthentik/prettier-config": "../prettier-config",
+        "@goauthentik/tsconfig": "../tsconfig",
```

Comment:
`*-config` packages are published without developer dependencies, allowing us to avoid a hard version here. This also fixes an timing issue when attempting to bump the version of multiple packages at the same time.